### DISCUSSION TITLE
use routing without mandatory gps

### DISCUSSION THREAD

**Discussion by macsimski** [osmandapp/OsmAnd#15164] [Ideas]
Is it possible to revert the program to the version where gps is not mandatory? I don't need it when planning my trip and announcing to the world at large where I am at that moment op planning is objectionable. Or is this an android 11 thing as I just switched from android 6 to 11.

**Comment by dmpr0** [comment]
You don't need GPS for planning routes in **Plan Route** mode, or when you want to preview the route in **Navigation** mode. GPS is needed for navigation.

**Reply by macsimski** [reply]
Oohh. I did'nt know that one. I always just set a marker and press navigation button in the lower right corner.

**Reply by dmpr0** [reply]
Yes, when you press the **Start** button in Navigation, it will always use your current location as a start, then navigate you to the first point you have set and then to the destination. So if you want to preview the route in **Navigation** mode, don't press **Start**.

**Comment by macsimski** [comment]
Ok after some testing, i just found out that giving an app the rights to access gps says nothing about the availability of gps. If you leave gps off. It will not use it.
Sorry. Migrating from android 6 to android 11 had some nasty road bumps for me.

Thanks for your support!

29 Aug 2022 08:45:45 Dmytro Prodchenko ***@***.***>:
